R/shared-methods.R

```diff
@@ -158,7 +161,7 @@ DataClass <- R6::R6Class(
     #' @field totals Boolean. If TRUE, returns totalled data per region
     #' up to today's date.
     totals = NULL,
-    #' @field localise Boolean. Should region names be localised.
+    #' @field localise Boolean. Should region names be localized.
     localise = NULL,
     #' @field verbose Boolean. Display information at various stages.
     verbose = NULL,
```

Comment:
These fields also I think should be private, or thought of and documented as being private or read-only, since they are typically set by the base initializer and then only used on a read-only basis.